### ISSUE TITLE
Bluethooth and icons orders 

### ISSUE THREAD

**Issue by farhatizakaria** [elementary/wingpanel#472]
### Problem

Well, when I disable bluetooth, I can't hide it from right click list neither from system settings. Also I can't ordering the other left icons or hide some of unwanted icons

### Proposal

More customization option for the panel from right click lists or from system settings :smile

### Prior Art (Optional)

We can use the way of XFCE panel customization, more options, more flexibility for a friendly experience

**Comment by peteruithoven** [comment]
Can you clarify what you mean with right click list? 

You're probably just referring to the icons on the right side of the panel? 

>Also I can't ordering the other left icons
Icons on the left? Applications is the only one on the left.

**Comment by farhatizakaria** [comment]
Hello @peteruithoven , I apologize if I haven't clarify my issue I have meant the indicator on the right side of the panel, I don't find how to hide Bluetooth icon or at least re-order them.
![Screenshot from 2023-01-28 15-47-36](https://user-images.githubusercontent.com/52662349/215273039-2abd61ee-bb33-45dd-a626-45965eed6d58.png)

**Comment by peteruithoven** [comment]
And you're mostly frustrated by the Bluetooth icon? Probably because you don't use Bluetooth (often)? 

Why would you want to re-order the icons? 

I assume the simplest solution would be adding a switch like the ones to show Caps or Num Lock in the panel in the Keyboard settings  to the Bluetooth settings. 
![Screenshot from 2022-07-21 11-04-59@2x-annotated](https://user-images.githubusercontent.com/523210/215275961-1a2dac18-2787-47a9-b603-88b7ec64171d.png)

**Comment by farhatizakaria** [comment]
Yeah I don't use it usually. Also, I find out that when my laptop locked, bluetooth was enabled automatically although I disabled it. So I think at least I have to choose between hiding the indicator or re-order it as in Xfce Panel for a balance between the enabled indicators and the disabled ones. But finally it's up to the developers 💁🏼 

![image](https://user-images.githubusercontent.com/52662349/215296661-5884af90-6fc7-4fea-b001-028533ec9996.png)

**Comment by stan-janssen** [comment]
I agree that reordering or selecting visibility might be useful if we can find a good spot for it.

In the meantime, there are a few different options for you.

### 1. Permanently disable bluetooth (also hides the Wingpanel Indicator), also solves the "Bluetooth is turned on after wake" issue:

```
sudo systemctl disable bluetooth
```

If you ever want to enable bluetooth again you can use:

```
sudo systemctl enable /lib/systemd/system/bluetooth.service
```

### 2. Remove the Wingpanel Indicator

```
sudo apt remove wingpanel-indicator-bluetooth
```

And when you need it again just:

```
sudo apt install wingpanel-indicator-bluetooth
```


(You can bookmark this comment for future reference if neccessary ;-))
